### PR TITLE
Add RNA Transcription exercise

### PR DESCRIPTION
**Pull request by houli**
Mostly followed the current exercises, in particular this PR #44. I see that some of the `config.json` keys have changed since that pull request so this will probably need some changes there.

### CODE REVIEW

**Comment by paf31** [comment]
Hmm looks like Travis is timing out. @lpil Any ideas?

File: exercises/rna-transcription/bower.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "rna-transcription",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "output"
+  ],
+  "dependencies": {
+    "purescript-console": "^3.0.0",
```
Comment (paf31) [inline exercises/rna-transcription/bower.json]: I don't think you need all of these.

File: exercises/rna-transcription/examples/src/RNATranscription.purs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  ( toRNA
+  ) where
+
+import Prelude
+import Data.Maybe (Maybe(..))
+import Data.String (fromCharArray, toCharArray)
+import Data.Traversable (traverse)
+
+toRNA :: String -> Maybe String
+toRNA dna = fromCharArray <$> traverse complement (toCharArray dna)
```
Comment (paf31) [inline exercises/rna-transcription/examples/src/RNATranscription.purs]: Nice!

File: exercises/rna-transcription/bower.json (hunk 1 above)
Comment (houli) [reply exercises/rna-transcription/bower.json]: Right, not sure what the deal is with the individual bower json files, the README isn't so clear. All of the exercises seem to have the same set of dependencies

Comment (lpil) [reply exercises/rna-transcription/bower.json]: I believe all the exercises share the same deps now so that we can cache easier on CI

Comment (paf31) [reply exercises/rna-transcription/bower.json]: Oh....

Ok, ignore me then.

**Comment by paf31** [comment]
If you could please resolve conflicts here, I'll get this merged. Thanks!

File: exercises/rna-transcription/bower.json (hunk 1 above)
Comment (icyrockcom) [reply exercises/rna-transcription/bower.json]: @houli Can you open an issue with details about what was unclear in the README, so I submit a PR to clear things up for future contributors?

@paf31 Yes, recently changed, details in: https://github.com/exercism/purescript/pull/71, https://github.com/exercism/purescript/pull/72 and https://github.com/exercism/purescript/pull/73

**Comment by paf31** [comment]
Thanks!

**Pull request merged**